napari/_qt/qt_main_window.py

```diff
@@ -137,9 +137,14 @@ def _add_file_menu(self):
         )
         open_folder.triggered.connect(self.qt_viewer._open_folder)
 
+        screenshot = QAction('Screenshot', self._qt_window)
+        screenshot.setStatusTip('Take screenshot')
```

Comment:
Can the status tip maybe be `Save screenshot` and the QAction be `Save Screenshot` or do you think that is implied / redundant?